web/themes/custom/itkdev/itkdev_project_theme/templates/reports-project-track.html.twig

```diff
@@ -30,14 +30,15 @@
     refused: 'bg-red-700',
   }
   %}
+
   {% set projectCount = data.projectTracks|length %}
```

Comment:
Is this another `projectCount` than the one you just defined in `AiScreeningReportsController::projectTrack`?